libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Identity/SFIdentityData.m

```diff
@@ -66,7 +66,7 @@
 NSString * const kSFIdentityCustomPermissionsKey          = @"custom_permissions";
 NSString * const kSFIdentityLastModifiedDateKey           = @"last_modified_date";
 
-NSString * const kSFIdentityDateFormatString              = @"yyyy-MM-dd'T'HH:mm:ss.SSSZZZ";
+NSString * const kSFIdentityDateFormatString              = @"yyyy-MM-dd'T'HH:mm:ssZZZ";
```

Comment:
🎉  This is was really bothering me and I was going to fix it today.  Thanks @wmathurin !